### PR TITLE
Add update task

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@fiji-flo r?

File: git/bin/update.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+pushd $1
+git pull origin master
```
Comment (akatsoulas) [inline git/bin/update.sh]: Maybe you should checkout master first before pulling.

Comment (MichaelKohler) [reply git/bin/update.sh]: yeah, that's actually a very good point! will adapt!

Comment (MichaelKohler) [reply git/bin/update.sh]: done

**Pull request closed without merging**